core/src/style/style.h

```diff
@@ -131,8 +108,13 @@ class Style {
     /* Add layers to which this style will apply */
     void addLayer(std::shared_ptr<SceneLayer> _layer);
```

Comment:
Style::addLayer should be redundant now. Also Style::m_layers.

Styles basically now have no ownership of sceneLayer/dataLayer.